### DISCUSSION TITLE
Installation Error

### DISCUSSION THREAD

**Discussion by ArdieWijaya** [vercel/next.js#41985] [Help]
Help I can't install the Next.js Application using the command provided in the Next.js Learning Guide
`npx create-next-app nextjs-blog --use-npm --example "https://github.com/vercel/next-learn/tree/master/basics/learn-starter"`

Node.js Version 14.17.5
macOS 12.6

Here is the error message
`❯ npx create-next-app nextjs-blog --use-npm --example "https://github.com/vercel/next-learn/tree/master/basics/learn-starter"
Creating a new Next.js app in /Users/ardiewijaya/Downloads/nextjs-blog.

Downloading files from repo https://github.com/vercel/next-learn/tree/master/basics/learn-starter. This might take a moment.

Installing packages. This might take a couple of minutes.

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: undefined@undefined
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   react@"17.0.2" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^18.0.0-0" from next@13.0.0
npm ERR! node_modules/next
npm ERR!   next@"latest" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /Users/ardiewijaya/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/ardiewijaya/.npm/_logs/2022-10-27T15_12_25_782Z-debug-0.log

Aborting installation.
  npm install has failed.`

[2022-10-27T15_12_25_782Z-debug-0.log](https://github.com/vercel/next.js/files/9880788/2022-10-27T15_12_25_782Z-debug-0.log)

Thanks for the help!

**Comment by relaxedtomato** [comment]
try updating your dependencies:
```
  "dependencies": {
    "next": "latest",
    "react": "latest",
    "react-dom": "latest"
  }
  ```
  
  `npm run dev` should run fine after
